test/templates/popover/index.njk

```diff
@@ -1,13 +1,22 @@
 {% extends "layout.njk" %}
 {% import "./../utils.njk" as utils %}
 {% from "./../format.njk" import format %}
-{% from "./component.njk" import popover, popover_popperjs %}
+{% from "./component.njk" import popover, popover_popperjs, list %}
 {% from "./../menu/component.njk" import menu %}
 
 {% block content %}
 
 <style media="screen">
     .fd-popover{ margin-left: 100px; margin-right: 100px;}
+
+    .popover-body__pgd {
```

Comment:
I think this should should be renamed to _not_ include "body" in the class name.  There is already a `fd-popover__body` class that defines the body container of the popover, but this class is really specific to the content that goes in that body.  It is also local to this playground file.  Maybe something like `.sample-content`?